sdk/storage/storage-queue/src/QueueServiceClient.ts

```diff
@@ -41,11 +42,36 @@ export interface ServiceSetPropertiesOptions {
    * about request cancellation.
    *
    * @type {Aborter}
-   * @memberof AppendBlobCreateOptions
+   * @memberof ServiceSetPropertiesOptions
    */
   abortSignal?: Aborter;
 }
 
+export type QueueServiceProperties = Models.StorageServiceProperties;
+
+/**
+ * Contains response data for the getProperties operation.
+ */
+export type ServiceGetPropertiesResponse = QueueServiceProperties & Models.ServiceGetPropertiesHeaders & {
```

Comment:
Why do we need put `QueueServiceProperties` into `ServiceGetPropertiesResponse` union type?

Reply:
I basically copied the definition of `Models.ServiceGetPropertiesResponse` here then replace the type `StorageServiceProperties` with the new alias type `QueueServiceProperties`.  Things should still work the same way because the two are the same type?

Reply:
So it depends on current swagger to work right? If the swagger changes, then the code won’t work